src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -158,3 +158,7 @@
 {% block javascripts %}
     {{ encore_entry_script_tags('ezplatform-admin-ui-location-view-js', null, 'ezplatform') }}
 {% endblock %}
+
+{% block meta %}
+    <meta name="LocationID" content="{{ location.id }}" />
```

Comment:
Seems like unused?

Reply:
It is used by the JS from admin-ui-modules https://github.com/ezsystems/ezplatform-admin-ui-modules/blob/88de2f7b8f806d0e90430a06523150c5aef63437/src/modules/universal-discovery/components/content-creator/content.creator.component.js#L33

Reply:
But this should be in special COTF template not in locationview.